addons/purchase_stock/wizard/product_replenish.py

```diff
@@ -34,7 +34,7 @@ def default_get(self, fields):
     def _compute_date_planned(self):
         super()._compute_date_planned()
         for rec in self:
-            if rec.route_id.name == 'Buy':
+            if rec.route_id.with_context(lang='en_US').name == 'Buy':
```

Comment:
even when forcing the language, using the name is super flimsy. What if a user renames this route or create a 'Buy for WH2' route?
```suggestion
            if 'buy' in rec.route_id.rule_ids.mapped('action'):
```